### ISSUE TITLE
AOP-Wiki queries do not return results

### ISSUE THREAD

**Issue by egonw** [VHP4Safety/cloud#60]
![image](https://github.com/VHP4Safety/cloud/assets/26721/0ae7960d-61b5-4b35-b1e8-50deb985658e)

(The Permalink button is also not working)

**Comment by egonw** [comment]
I believe the problem is the URL of the SPARQL endpoint, which states `vhp4safety.org` instead of `vhp4safety.nl`.

**Comment by ozancinar** [comment]
Yes, I agree. I think the environmental variable (see the SNORQL_ENDPOINT variable [here](https://github.com/VHP4Safety/wp1.1/blob/main/misc/service_deployment_with_docker_run/aopwiki.md)) was incorrect when the container was initiated. I'll try to correct it.

**Comment by ozancinar** [comment]
I tested it on another server (see [here](http://85.214.37.128:8083/)). The SPARQL endpoint seems to be correct now; however, the examples do not return results which is unexpected I presume. I suspect the reason for this is that there is a mismatch between the service version we deploy and the version the data sets that go along with the service. Note that we changed the data files when we were trying the new version of the service. I am not exactly sure if this is the issue, and if it I am not sure which versions I have. I will look more into this and discuss it with Marvin to check whether the data files are correct.
